### DISCUSSION TITLE
export faults as elements for pflotran

### DISCUSSION THREAD

**Discussion by anneschulz1991** [gempy-project/gempy#677] [Q&A]
Hello all,
I have recently started getting into gempy to create geological grids as input files for pflotran. For this I have already familiarized myself with the export function. However, I still have a problem. To assign properties to a fault in Pflotran, I need elements. Is there a way to create a formation that corresponds to the course of a fault. I could subsequently export the elements of this formation and define them as a fault in Pflotran.
I hope someone can help me. In any case, I thank you in advance for your efforts.
Anne

**Comment by Japhiolite** [comment]
Hi Anne,

sorry for the late reply. I'd have to look at the export function to pflotran, if fault vertices get used at all in that export function. Thing is, if it uses the regular grid of gempy, faults are not represented in there and thus won't show up as elements in pflotran. Making faults usable as "units" (to assign properties, when exported), is on the roadmap though.

Best,
Jan

**Comment by anneschulz1991** [comment]
Hello Jan,

that sounds good. How much time do you think it will take to add faults as a "unit"? I would really like to use gempy to generate my grids, but absolutely need this function. 
Best regards
Anne

**Reply by Japhiolite** [reply]
Hi Anne,

unfortunately, I cannot give a precise timing on that, as we're currently planning a rather complete overhaul of gempy. How is your pflotran model gridded? Do you use the regular grid from gempy?
Best
Jan
